types/react-table/index.d.ts

```diff
@@ -112,7 +112,7 @@ export type UseTableOptions<D extends object> = {
     defaultColumn: Partial<Column<D>>;
     initialRowStateKey: IdType<D>;
     getSubRows: (originalRow: D, relativeIndex: number) => D[];
-    getRowId: (originalRow: D, relativeIndex: number) => IdType<D>;
+    getRowId: (originalRow: D, relativeIndex: number, parent?: Row<D>) => string;
```

Comment:
I believe this should still return and `IdType<D>`?

Reply:
No, it returns an arbitrary identifier. The default implementation returns the row number as a string, or a concatenation of row and parent row for grouped records.

Reply:
No, it returns an arbitrary identifier. The default implementation returns the row number as a string, or a concatenation of row and parent row for grouped records.